02_CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+Before contributing to Lob-OpenAPI, kindly [create an issue](https://github.com/lob/lob-openapi/issues/new/choose) first, in order to explain your proposed change to the repository owners :godmode:. We welcome call-outs of typos, bad wording, or anything else which catches your eye!
+
+Please note we have a code of conduct. Ensure that you follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
```

Comment:
More a question for me but what's a "layer" in this context?

Reply:
This is from a template, so yeah the wording here is a little weird... I think they just mean "don't add unnecessary dependencies you've been experimenting with on your end". I can edit it quickly.